mm2src/mm2_main/src/rpc/lp_commands/lp_commands.rs

```diff
@@ -34,6 +34,7 @@ impl HttpStatusCode for GetPublicKeyError {
     }
 }
 
+// TODO: Return public_key for all available and active unique pubkeys
 pub async fn get_public_key(ctx: MmArc, _req: Json) -> GetPublicKeyRpcResult<GetPublicKeyResponse> {
```

Comment:
We can leave this as todo and do it in another separate PR, please add it to checklist. We still need to work out the details for the enhancements needed for `get_public_key` so that we can also get the public key of any path in the hd wallet or multiple pubkeys for an account, or the whole account pubkey, etc..
It can probably be a new RPC where we get the public key for a specific coin with additional optional params.

Reply:
P.S. we will need a similar get private key / `show_priv_key` v2 method for HD wallet.

Reply:
added to PR description